config/default.yml

```diff
@@ -860,7 +860,10 @@ Layout/FirstMethodArgumentLineBreak:
                  multi-line method call.
   Enabled: false
   VersionAdded: '0.49'
+  VersionChanged: '<<next>>'
   AllowMultilineFinalElement: false
+  AllowedMethods: []
+  AllowedPatterns: []
```

Comment:
Perhaps only `AllowedMethods` should be provided, and `AllowedPattern` might not be necessary.

Reply:
`AllowedPattern` would allow the original issue to be neatly resolved by matching all 1-character method names.

Reply:
Certainly, it is possible to patternize a single character, but since method names typically start with a lowercase letter and the number of single-character method names is limited, I think they can be listed in `AllowedMethods`.

Reply:
I think there are other cases that a regexp would be useful, and personally would prefer configuring `/^.$/` over a list of method names were I to have this requirement; however, I have removed the `AllowedPattern` configuration from the cop in the interest of getting this done.